generator.go

```diff
@@ -86,11 +88,6 @@ func (g Generator) Generate(ctx context.Context, root string, args []string) err
 	log.Infof("migration generated in %v", path)
 	err = g.addToChangelog(root, path)
 
-	if os.IsNotExist(err) {
```

Comment:
@paganotoni removed this one because we are making sure the changelog file always exist